tools/pom.xml

```diff
@@ -34,6 +34,8 @@
 
   <modules>
     <module>cmdline</module>
+    <module>maven-plugin</module>
+    <module>shared</module>
```

Comment:
Would you mind @scottmarlow if we wouldn't introduce 'shared' submodule inside tools folder?
My initial idea was maven plugin would depend and delegate to cmdline tool.

Reply:
To be more explicit, command line tool performs parameters validation and prints usage message
if parameters are misused. It would be nice to have parameters validation in just one place.
If we will introduce 'shared' submodule then parameters validation would need to exist
in both 'cmdline' and 'maven' tools and that would introduce some duplicities.

Reply:
Of course having parameters validation on multiple places isn't such a big problem and I don't insist on avoiding it. I just wanted to share with you my initial idea and discuss it with you first before proceeding with this proposal pull request @scottmarlow .